pkg/controller/replicaset/replica_set_utils.go

```diff
@@ -126,7 +126,7 @@ func calculateStatus(rs *extensions.ReplicaSet, filteredPods []*v1.Pod, manageRe
 	return newStatus
 }
 
-// NewReplicaSetCondition creates a new replica set condition.
+// NewReplicaSetCondition creates a new replicaset condition.
```

Comment:
but the kind: ReplicaSet ，I look the doc in https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
both use ReplicaSet objects